scripts/ci/linux/appimage.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+########################################################################
+# Package the binaries built on Travis-CI as an AppImage
+# By Simon Peter 2016
+# Modified for Moolticute by Alexis López 2017
+# For more information, see http://appimage.org/
+########################################################################
+
+
+export ARCH=$(arch)
+
+APP=moolticute
+LOWERAPP=${APP,,}
+
+BASE_PATH=$PWD/build-appimage/
+APPDIR=$BASE_PATH/$APP/$APP.AppDir/
+TARGET_DIR=$APPDIR/usr/
+
+mkdir -p $TARGET_DIR
+
+SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $SCRIPTDIR/appimage_functions.sh
+source $SCRIPTDIR/../funcs.sh
+
+########################################################################
+# Install application binaries to the AppDir
+########################################################################
+cp ./debian/moolticute/usr/* $TARGET_DIR -r
+
+########################################################################
+# Generate .desketop file and icon
+########################################################################
+
+echo "[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Moolticute
+Comment=Moolticute allows your browser extensions to talk with the Mooltipass.
+TryExec=moolticute
+Exec=moolticute %F
+Icon=moolticute" > $BASE_PATH/$APP/$APP.AppDir/moolticute.desktop
+
+wget -O $BASE_PATH/$APP/$APP.AppDir/moolticute.svg https://raw.githubusercontent.com/mooltipass/moolticute/master/img/AppIcon.svg
```

Comment:
Why downloading this? you have the checked out git source tree accessible directly